save-backend/src/main/kotlin/org/cqfn/save/backend/repository/JpaSpecificationExecutorRepository.kt

```diff
@@ -0,0 +1,14 @@
+package org.cqfn.save.backend.repository
+
+import org.springframework.data.jpa.domain.Specification
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor
+
+/**
+ * Extension interface for JpaSpecificationExecutor
+ */
+interface JpaSpecificationExecutorRepository<T> : JpaSpecificationExecutor<T> {
+    /**
+     * @param spec Specification.
+     */
+    fun getFirstResultOrNull(spec: Specification<T>) = this.findAll(spec).getOrNull(0)
```

Comment:
Will it perform `get` operation on client side, with all data fetched from DB? If so, I believe limiting results in DB will be more preferable, e.g. by passing a `Pageable` of single element to `findAll`